### PR TITLE
Document how new-app creates EmptyDir volumes for containers

### PR DESCRIPTION
**Pull request by mnagy**
@bparees PTAL

### CODE REVIEW

File: dev_guide/new_app.adoc [hunk 1]
```diff
@@ -388,7 +388,9 @@ image stream is created for that image as well.
 
 a|`*DeploymentConfig*`
 a|A `*DeploymentConfig*` is created either to deploy the output of a build, or a
-specified image.
+specified image. If the Deployment Config contains specifications for containers,
```
Comment (bparees) [inline dev_guide/new_app.adoc]: would a deploymentconfig ever not contain  a specification for containers?

Comment (mnagy) [inline dev_guide/new_app.adoc]: I guess not :) Good catch.

**Comment by mnagy** [comment]
PTAL

**Comment by bparees** [comment]
lgtm.  @adellape ?

**Comment by adellape** [comment]
@mnagy LGTM. It seems like this made it into OSE 3.0 GA; can you confirm? If it didn't, I'll conditionalize it out of the OSE docs for now.

**Comment by mnagy** [comment]
@adellape no it did not. It was included in 3.0.1, however.

**Pull request closed without merging**